src/lib/uri.ts

```diff
@@ -48,12 +49,18 @@ const handleAuthRequest = async (authParams: URLSearchParams) => {
     await createEmptyConfigFile({ path: configPath });
     await trackEvent(TRACKED_EVENTS.AUTH_INITIALIZE_CONFIG, { name });
   }
-
-  const updatedConfig = await updateConfigWithPersonalAccessKey({
+  let token;
+  try {
+    token = await getAccessToken(personalAccessKey, env);
+  } catch (err) {
+    throw err;
+  }
+  const updatedConfig = await updateConfigWithAccessToken(
+    token,
     personalAccessKey,
-    name,
     env,
-  });
+    name
+  );
```

Comment:
`updateConfigWithPersonalAccessKey` no longer exists in LDL so fetching token first and then using `updateConfigWithAccessToken`